lib/features/base/base_controller.dart

```diff
@@ -471,4 +472,12 @@ abstract class BaseController extends GetxController
     authorizationInterceptors.clear();
     await cachingManager.closeHive();
   }
+
+  int getMinInputLengthAutocomplete({
+    required Session session,
+    required AccountId accountId,
+  }) {
+    final minInputLength = session.getMinInputLengthAutocomplete(accountId);
+    return minInputLength?.value.toInt() ?? AppConfig.defaultMinInputLengthAutocomplete;
+  }
```

Comment:
Why need to put it in controller? How about to use extension or util class?

Reply:
We already had handle `getMinInputLengthAutocomplete` in session extension and wirte unit test for it. This function to get value default `minInputLengthAutocomplete` when minInputLength from session is null. Additional, we only use it on all controller.